kogito-scripts-quarkus/README.md

```diff
@@ -69,3 +81,12 @@ Complete curl command can be found below:
 ```
 curl -X POST -H 'Content-Type:application/json' -H 'Accept:application/json' -d '{"name" : "john"}' http://localhost:8080/scripts
 ```
+
+Log after curl executed
+```
+{"id":"ab5239e2-f497-4684-b337-5a44440b38dd","name":"john","message":"Hello john"} 
+```
+
+Likewise on Quarkus you should see a similar log to
+
+<p align="center"><img src="docs/images/quarkusLog.png"></p>
```

Comment:
The screenshot is hardly visible for the log ...
Would it be possible to put the logs as panel text instead of log screenshot ?

Reply:
fixed @radtriste